AUTHORS.md

```diff
@@ -1,17 +1,15 @@
-Main Contributors
-=================
-
-Main contributors to code and ideas in alphabetical order:
-----------------------------------------------------------
-
-* Mike Gevaert @ BBP
-* Lida Kanari @ BBP
-* Juan Palacios @ BBP
-* Eleftherios Zisis @ BBP
-
-Maintainer / release manager
-----------------------------
-
-* [Benoit Coste @ BBP](https://github.com/wizmer)
-* [Michael Gevaert @ BBP](https://github.com/mgeplf)
+Maintainer
+----------
+* Michael Gevaert
+
+Authors
+-------
+* Benoit Coste
+* Mike Gevaert
+* Lida Kanari
+* Juan Palacios
+* Eleftherios Zisis
+
+Contributors
```

Comment:
What's the point of empty contributors?

Reply:
Just to show that there is such a section (for presentation purposes). Akiko will use that file as an example on how the AUTHORS file should be formatted in the future.

Reply:
ok